src/riak_kv_pb_object.erl

```diff
@@ -379,6 +434,38 @@ process_stream(_,_,State) ->
 %% Internal functions
 %% ===================================================================
 
+-spec make_binarykey(riak_object:riak_object()) -> binary().
+make_binarykey(RObj) ->
+    make_binarykey(riak_object:bucket(RObj), riak_object:key(RObj)).
+
+-spec unpack_keyclock_fun(#rpbkeysvalue{}) ->
+        {riak_object:bucket(RObj), riak_object:key(RObj), vclock:vclock(),
+            to_fetch}.
+unpack_keyclock_fun(RpbKeysClock) ->
+    case RpbKeysClock#rpbkeysvalue.type of
```

Comment:
This is code duplication. I wonder if it would not be easier to write 
```suggestion
    Bucket = case RpbKeysClock#rpbkeysvalue.type of
                             undefined -> RpbKeysClock#rpbkeysvalue.bucket;
                             T -> {T, RpbKeysClock#rpbkeysvalue.bucket}
                      end,
       {Bucket, RpbKeysClock#rpbkeysvalue.key,
                 riak_object:decode_vclock(RpbKeysClock#rpbkeysvalue.value),
                 to_fetch}
```

Reply:
Agreed, will change